Makefile

```diff
@@ -155,6 +152,22 @@ else
 KIND=$(shell which kind)
 endif
 
+setup-envtest:
+ifeq (, $(shell which setup-envtest))
+	@{ \
+	set -e ;\
+	ENVTEST_GEN_TMP_DIR=$$(mktemp -d) ;\
+	cd $$ENVTEST_GEN_TMP_DIR ;\
+	go mod init tmp ;\
+	GO111MODULE="on" go get sigs.k8s.io/controller-runtime/tools/setup-envtest@latest ;\
```

Comment:
Can we use `go install` and save the creation of the temp dir maybe?

Reply:
Sure, I've changed that and specified a version to be used as well